pkg/testgridanalysis/testreportconversion/by_tests.go

```diff
@@ -21,6 +21,35 @@ func getTopFailingTestsWithoutBug(testResultsByName testResultsByName, testResul
 	}, testResultFilterFn)
 }
 
+// curatedTestSubstrings is keyed by release.  This is a list of tests that are important enough to individually watch.
+// Whoever is running or working on TRT gets freedom to choose 10-20 of these for whatever reason they need.  At the moment,
+// we're chasing problems where pods are not running reliably and we have to track it down.
+var curatedTestSubstrings = map[string][]string{
+	"4.6": []string{
+		"[Feature:SCC][Early] should not have pod creation failures during install",
+		"infrastructure should work",
+		"install should work",
+		"Kubernetes APIs remain available",
+		"OAuth APIs remain available",
+		"OpenShift APIs remain available",
+		"Pod Container Status should never report success for a pending container",
+		"pods should never transition back to pending",
+		"pods should successfully create sandboxes",
+		"upgrade should work",
```

Comment:
you've introduced a couple of "curated lists" of things at this point, can you reconcile them all into a single file so that when the next TRT team takes over and wants to change this, it's all in one obvious location?

i'm thinking of things like `customJobSetupContainers` and `jobsNeverStableForPlatforms`

(long term this should probably all become config yaml, in an ideal world, but i'm realistic)